beater/test_approved_es_documents/TestPublishIntegrationMetricsets.approved.json

```diff
@@ -303,6 +303,69 @@
                 "id": "axb123hg",
                 "name": "logged-in-user"
             }
+        },
+        {
+            "@timestamp": "2017-05-30T18:53:41.369Z",
+            "agent": {
+                "name": "elastic-node",
+                "version": "3.14.0"
+            },
+            "aws.lambda.metrics.billed_duration": 183,
+            "aws.lambda.metrics.coldstart_duration": 422.9700012207031,
+            "aws.lambda.metrics.duration": 182.42999267578125,
+            "aws.lambda.metrics.timeout": 5000,
+            "data_stream.dataset": "apm.internal",
+            "data_stream.namespace": "default",
+            "data_stream.type": "metrics",
+            "ecs": {
+                "version": "1.12.0"
+            },
+            "faas": {
+                "coldstart": true,
+                "execution": "6f7f0961f83442118a7af6fe80b88d56",
+                "id": "arn:aws:lambda:us-east-2:123456789012:function:custom-runtime"
```

Comment:
I had to double check, but it seems that the Extensions API [does not provide the Layer version](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-extensions-api.html) - here that would be
```
arn:aws:lambda:us-east-2:123456789012:function:custom-runtime:1
```
That information would be useful to have, but there is not much we can do for now that improve that I think.